src/CsvTextEditor/Constants.cs

```diff
@@ -37,6 +37,11 @@ public static class AutomaticUpdates
     public static class Configuration
     {
         public const string CustomEditor = "Settings.Application.Editor.CustomEditor";
+        public const string TimedAutoSave = "Settings.Application.Editor.AutoSaveEditor";
```

Comment:
Either the constant name or the literal is incorrect, shouldn't it be AutoSaveTime or something?

Reply:
AutoSaveInterval